src/dependencies.stanza

```diff
@@ -5,38 +5,62 @@ defpackage slm/dependencies:
   import maybe-utils
   import semver
   import term-colors
-  import toml/file
-  import toml/parser
-  import toml/table
+  import toml
 
   import slm/flags
-  import slm/dependency
+  import slm/dep-types
```

Comment:
thought you implemented dependent types for a sec....

Reply:
Ha - I'm not that ambitious.